ipaserver/install/kra.py

```diff
@@ -47,9 +48,7 @@ def install_check(api, replica_config, options):
 
     if replica_config is not None:
         if not api.Command.kra_is_enabled()['result']:
-            raise RuntimeError(
-                "KRA is not installed on the master system. Please use "
-                "'ipa-kra-install' command to install the first instance.")
+            return
```

Comment:
We may want to log a (info level) comment here that no KRA was found so we are going to install a new one from scratch.